frame/staking/src/pallet/impls.rs

```diff
@@ -599,20 +599,19 @@ impl<T: Config> Pallet<T> {
 
 	/// Apply previously-unapplied slashes on the beginning of a new era, after a delay.
 	fn apply_unapplied_slashes(active_era: EraIndex) {
-		let slash_defer_duration = T::SlashDeferDuration::get();
-		<Self as Store>::EarliestUnappliedSlash::mutate(|earliest| {
```

Comment:
as far as I can see, we no longer use `EarliestUnappliedSlash` storage - should we get rid of it?

Reply:
absolutely, I actually had removed it, got somewhere lost in undo-redo-ing process while testing.